files/configs/event-router/event-router-clusterrole.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: veba-addressable-resolver
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: addressable-resolver
```

Comment:
just note that this will change with more recent knative versions, e.g. see https://github.com/vmware-samples/vcenter-event-broker-appliance/blob/99d0139b3745d25a58e15391a2593690645e6043/vmware-event-router/chart/templates/100_clusterrolebinding.yaml#L11